app-web/registry/community-and-events.json

```diff
@@ -50,19 +53,33 @@
       },
       {
         "sourceType": "github",
+        "resourceType": "Self-Service Tools",
+        "sourceProperties": {
+          "resourceType": "Documentation",
```

Comment:
i think you may want to stick that up one line, resource type is not in the source properties. 

> i know we need to fix how we mark up the registry :sweat: